Solutions/Corvus.Extensions.Specs/CollectionExtensionsFeatures/CollectionExtensionsSteps.cs

```diff
@@ -0,0 +1,95 @@
+﻿// <copyright file="CollectionExtensionsSteps.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Corvus.Extensions.Specs.CollectionExtensionsFeatures
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using NUnit.Framework;
+    using TechTalk.SpecFlow;
+
+    [Binding]
+    public class CollectionExtensionsSteps
+    {
+        private ICollection<int>? destination;
+        private List<int>? source;
+        private Exception? exception;
+
+        [Given(@"I have a destination list containing (.*)")]
```

Comment:
Please can we remove the verbatim string literals (`@` symbols) as they aren't needed here.

If you install Roslynator, it will flag these and give you a quick fix for them.